lib/app.dart

```diff
@@ -194,7 +192,7 @@ class HyphaAppView extends StatelessWidget {
             theme: HyphaTheme.lightTheme,
             themeMode: state.themeMode,
             navigatorObservers: <NavigatorObserver>[observer],
-            home: const SizedBox.shrink(),
+            home: const OnboardingPage(),
```

Comment:
Initial page is onboarding.

Reply:
but if initial page is onboarding, don't we lose the "import account" page?

Since onboarding doesn't even have a close or back or exit button...